modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {KialiProduct}
```

Comment:
The product name is really `Kiali Operator by Red Hat`?

Reply:
I had the same thought and mentioned it below in my own comment. It does result in some odd phrases that aren't technically correct.

Reply:
Yep. In the UI, users can either select "Kiali Operator provided by Red Hat" or the Kiali Community Operator. For prod docs, it is "Kiali Operator provided by Red Hat."